paper/paper.md

```diff
@@ -11,49 +14,57 @@ authors:
 
-Compared to other solutions [@mlflow; @wandb], `PyExperimenter` is very lightweight and has only a handful of dependencies. Furthermore it is designed to support simple but effective configurations. 
+## Statement of Need
+The recent advances in artificial intelligence have uncovered a need for experiment tracking functionality, leading to the emergence of several tools addressing this issue. Prominent representatives include Weights and Biases [@wandb], MLFlow [@mlflow], TensorBoard [@tensorboard], neptune.ai [@neptune], Comet.ML [@comet], Aim [@aim], Data Version Control [@dvc], Sacred [@sacred], and Guild.AI [@guildai]. These tools largely assume that users define the configuration of an experiment together with the experiment run itself. In case of the evaluation of different hyperparameter configurations, this process is suboptimal, since it requires to communicate the hyperparameters through scripts. This task can become cumbersome to manage as the number of configuration options and desired combinations grows and becomes more complex. Weights and Biases [@wandb], Polyaxon [@polyaxon], and Comet.ML [@comet] allow so-called sweeps, i.e., hyperparameter optimization, albeit in a limited way. For a sweep, usually hyperparameters that should be optimized are specified along with the desired search domains, and an optimizer can be selected from a pre-defined list to carry out the optimization. However, the implementation of this functionality usually imposes several restrictions on the way the sweep can be carried out.
 
+In contrast, `PyExperimenter` follows an inverted workflow. Instead of experiment runners registering experiments to a tracking entity such as a tracking server or database, the experiments are predefined and runners are pulling open experiments from a database. Similarly, ClearML [@clearml] and Polyaxon [@polyaxon] support a more generic workflow where experiments are first enqueued in a central orchestration server and agents can then pull tasks from the queue to execute them. However, both are much more heavyweight than `PyExperimenter`, regarding the implementation of both the agents and backend-features, and are further neither completely open-source nor free.
```

Comment:
Similarly, ClearML [@clearml] and Polyaxon [@polyaxon] support a more generic workflow where experiments are first enqueued in a central orchestration server and agents can then pull tasks from the queue to execute them. However, both are much more heavyweight than `PyExperimenter`, regarding the implementation of both the agents and backend-features, and are further neither completely open-source nor free. ->Similarly, ClearML [@clearml] and Polyaxon [@polyaxon] support a more generic workflow where experiments are first enqueued in a central orchestration server and agents can then pull tasks from the queue to execute them. However, both are much more heavyweight than `PyExperimenter`, regarding the implementation of both the agents and backend-features. Further, they are neither completely open-source nor free.

I had to read the sentence a few times to get the syntax. I recommend splitting it

Reply:
I think the comma before "regarding" should be left out.
Instead of "Further", I would suggest "Moreover" or "Furthermore". Also I would switch the order of "completely open-source" and "free". Sounds somehow better to me.

Reply:
I agree with Marcel except for the last point which I don't really care about either way.